src/content/wiki/items/tag_tool.mdoc

```diff
@@ -60,16 +60,21 @@ The following tags are currently implemented (more may be added from time to tim
+    | ---------------| ----------------------------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+    | `sit`          | {% building name="cook" /%}         | 0-many      | A "chair" where colonists will sit to eat their food                                                                                      |
+    | `sit`          | {% building name="tavern" /%}       | 0-many (~4) | A "chair" where visitors will occasionally sit                                                                                            |
+    | `sit`          | (leisure)                           | 0-many      | A "chair" where colonists will occasionally sit                                                                                           |
+    | `cobble`       | {% building name="miner" /%}        | Always 1    | The block behind the top-most ladder at the start of the mineshaft (traditionally cobblestone, but may be something else)                 |
+    | `ladder`       | {% building name="miner" /%}        | Always 1    | The top-most ladder at the start of the mineshaft                                                                                         |
+    | `bamboo`       | {% building name="plantation" /%}   | level-based | A block that bamboo can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                            |
+    | `cactus`       | {% building name="plantation" /%}   | level-based | A block that cactus can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                            |
+    | `sugar`        | {% building name="plantation" /%}   | level-based | A block that sugarcane can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                         |
+    | `portal`       | {% building name="netherworker" /%} | Always 1    | An obsidian block in the bottom of the nether portal (with air above)                                                                     |
+    | `vault`        | {% building name="netherworker" /%} | 0-1         | A block in an enclosed room that the nether miner can stand on while "away"                                                               |
+    | `work`         | {% building name="archery" /%}      | level-based | The block the {% worker name="archer" /%} in Training stands on to shoot at targets (Note: If no tags are used, they look for glowstone.) |
+    | `job=miner`    | {% building name="mine" /%}         | Always 1    | Sets the mine's worker to be only {% worker name="miner" /%}                                                                              |
```

Comment:
Building is miner, not mine

Reply:
I will not, you want this in, you fix it, besides you already put the building I'd correctly elsewhere

Reply:
ok for a start, I don't know how to do it without going back and redoing the entire thing. I also don't know if the formatting is right as table sizing changed. this is why I liked it when Rentsuru was in charge of the wiki as he would fix it for me after I corrected the information as formatting etc is above my understanding. there is NO need for you to be antagonistic on this mate.